### PR TITLE
Private keys and passwords present.

### PR DESCRIPTION
**Pull request by the4rchangel**

### CODE REVIEW

File: tuf/utils/pkcs8_test.go [hunk 1]
```diff
@@ -11,33 +11,7 @@ import (
 )
 
 func getRSAKey() (data.PrivateKey, error) {
-	raw := []byte(`-----BEGIN RSA PRIVATE KEY-----
```
Comment (thaJeztah) [inline tuf/utils/pkcs8_test.go]: These keys and passwords are specifically created for these tests, so it's ok to have them in here

Comment (HuKeping) [reply tuf/utils/pkcs8_test.go]: it seems no risk here

Comment (thaJeztah) [reply tuf/utils/pkcs8_test.go]: yes, I think this should be ok to close, unless we want to add a comment in files to explain this

**Comment by HuKeping** [comment]
closing.

Please feel free to reopen it if there is any further work need to do.

**Pull request closed without merging**